test/functionalTest/cases/3121_missing_or_empty_metadata_values/entitiy_creation.test

```diff
@@ -0,0 +1,408 @@
+payload='{
+  "id": "Room2",
+  "type": "Room",
+  "temperature": {
+    "value": 26.5,
+    "type": "Float",
+    "metadata": {
+      "accuracy": {
+        "value": "",
+        "type": "Float"
+      },
+     "tolerance": {
+        "type": "Float"
+      }
+    }
+  }
+}'
+orionCurl --url /v2/entities --payload "$payload"
+echo
+echo
+
+
+echo "08. GET /v2/entities/Room2."
+echo "==========================="
+orionCurl --url /v2/entities/Room2
+echo
+echo
+
+
+echo "09. PUT /v2/entities/Room2/attrs/temperature/value, updating the value of Room2 temperature attribute."
+echo "======================================================================================================"
+payload='{
+  "value": 20.6,
+  "type": "Number",
+  "metadata": {
+    "accuracy": {
+      "value": "",
+      "type": "Float"
+    },
+    "tolerance": {
+      "type": "Float"
+    }
+  }
+}'
+ orionCurl --url /v2/entities/Room2/attrs/temperature --payload "$payload" -X PUT 
+echo
+echo
+
+
+echo "10. GET /v2/entities/Room2."
+echo "==========================="
+orionCurl --url /v2/entities/Room2
+echo
+echo
+
+
+--REGEXPECT--
+01. POST /v2/entities, to create Room1 with temperature (no metadata).
+======================================================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/entities/Room1?type=Room
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+02. GET /v2/entities/Room1.
+===========================
+HTTP/1.1 200 OK
+Content-Length: 84
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "id": "Room1",
+    "temperature": {
+        "metadata": {},
+        "type": "Float",
+        "value": 23
+    },
+    "type": "Room"
+}
+
+
+03. POST /v2/entities/Room1/attrs, to add metadata with  value to temperature and add pressure attribute with metadata with no value.
+=======================================================================================================================================
+HTTP/1.1 204 No Content
+Content-Length: 0
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+04. GET /v2/entities/Room1.
+===========================
+HTTP/1.1 200 OK
+Content-Length: 218
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "id": "Room1",
+    "pressure": {
+        "metadata": {
+            "accuracy": {
+                "type": "Float",
+                "value": null
+            }
+        },
+        "type": "Integer",
+        "value": 720
+    },
+    "temperature": {
+        "metadata": {
+            "accuracy": {
+                "type": "Float",
+                "value": ""
+            }
+        },
+        "type": "Float",
+        "value": 25
+    },
+    "type": "Room"
+}
+
+
+05. POST /v2/entities/Room1/attrs, to update temperature with metadata with no value, pressure with metadata with regular value and add humidity attribute with metadata with  value.
```

Comment:
Better use `echo '...'` when you have `"` in the message you want to print (as in this case and some others in the tests). In addition, also as general comment, try that the line with `=` have the same length that the message printed above.

Reply:
fixed in bcf8353